bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/discovery/ThingDiscoveryService.java

```diff
@@ -59,31 +66,22 @@ public class ThingDiscoveryService extends AbstractDiscoveryService implements T
             new AbstractMap.SimpleEntry<>("TWINGUARD", BoschSHCBindingConstants.THING_TYPE_TWINGUARD),
             new AbstractMap.SimpleEntry<>("PSM", BoschSHCBindingConstants.THING_TYPE_INWALL_SWITCH), // Steckdosen Zwischenstecker
             new AbstractMap.SimpleEntry<>("PLUG_COMPACT", BoschSHCBindingConstants.THING_TYPE_SMART_PLUG_COMPACT),
-            new AbstractMap.SimpleEntry<>("CAMERA_EYES", BoschSHCBindingConstants.THING_TYPE_CAMERA_360),
-            new AbstractMap.SimpleEntry<>("BWTH", BoschSHCBindingConstants.THING_TYPE_WALL_THERMOSTAT), // floorheating room control
+            new AbstractMap.SimpleEntry<>("CAMERA_350", BoschSHCBindingConstants.THING_TYPE_CAMERA_360),
+            new AbstractMap.SimpleEntry<>("CAMERA_EYES", BoschSHCBindingConstants.THING_TYPE_CAMERA_EYES),
+            new AbstractMap.SimpleEntry<>("BWTH", BoschSHCBindingConstants.THING_TYPE_THERMOSTAT), // wall thermostat
+            new AbstractMap.SimpleEntry<>("THB", BoschSHCBindingConstants.THING_TYPE_WALL_THERMOSTAT), // wall thermostat with batteries
             new AbstractMap.SimpleEntry<>("SD", BoschSHCBindingConstants.THING_TYPE_SMOKE_DETECTOR),
             new AbstractMap.SimpleEntry<>("MD", BoschSHCBindingConstants.THING_TYPE_MOTION_DETECTOR),
             new AbstractMap.SimpleEntry<>("ROOM_CLIMATE_CONTROL", BoschSHCBindingConstants.THING_TYPE_CLIMATE_CONTROL),
             new AbstractMap.SimpleEntry<>("INTRUSION_DETECTION_SYSTEM", BoschSHCBindingConstants.THING_TYPE_INTRUSION_DETECTION_SYSTEM),
-            new AbstractMap.SimpleEntry<>("SMOKE_DETECTION_SYSTEM", BoschSHCBindingConstants.THING_TYPE_SMOKE_DETECTOR),
-            new AbstractMap.SimpleEntry<>("HUE_LIGHT", BoschSHCBindingConstants.THING_TYPE_SMART_BULB)
-
-// TODO: map all BoschSHC Thing Types to a deviceModel name
-//
-
-//            new AbstractMap.SimpleEntry<>("???", BoschSHCBindingConstants.THING_TYPE_WINDOW_CONTACT),
-//            new AbstractMap.SimpleEntry<>("???", BoschSHCBindingConstants.THING_TYPE_THERMOSTAT),
-//            new AbstractMap.SimpleEntry<>("???", BoschSHCBindingConstants.THING_TYPE_CAMERA_EYES),
-//            new AbstractMap.SimpleEntry<>("???", BoschSHCBindingConstants.THING_TYPE_SMART_BULB),
-
-// TODO: map all deviceModel names from the openhab logs to BoschSHC Thing Types
-//
-
+            new AbstractMap.SimpleEntry<>("HUE_LIGHT", BoschSHCBindingConstants.THING_TYPE_SMART_BULB),
```

Comment:
This is for Ledvance Smart Bulbs that connect via Zigbee directly to the bridge. If I'm not mistaken it has nothing do do with the Hue system.

Reply:
Hmm, this works for me:
The Hue light bulb is connected to a Hue bridge. The Hue bridge is added to the SHC bridge.
I can control a "Smart BULB" light in openhab in the same way as a light via the Bosch SHC App.